# Awesome Phylogenetic Inference
A curated list of resources about Phylogenetics Inference.

## Table of Contents
- [Introduction to Phylogenetics](#introduction_to_phylogenetics)
- [Books and Lectures](#books_and_lectures)
- [Papers](#papers)
- [Databases](#databases)
- [Methods](#methods)
- [Computational Tools](#computational_tools)

<a name="introduction_to_phylogenetics" />

## Introduction to Phylogenetics
Phylogenetics inference is the process of reconstructing the evolutionary relationships among species or genes based on various types of data, predominantly molecular sequences, morphological traits, or a combination thereof. At its essence, this field aims to unveil the intricate branching patterns of life, tracing the common ancestry and evolutionary divergence between organisms. Through sophisticated computational algorithms and statistical models, phylogenetics inference endeavors to construct phylogenetic trees that represent the most likely evolutionary history of the entities under study. These trees serve as evolutionary blueprints, depicting the hierarchical relationships and evolutionary distances between taxa or genetic sequences, providing crucial insights into the evolutionary processes that have shaped biodiversity.

The methodologies within phylogenetics inference range from classical approaches like distance-based and parsimony methods to more intricate probabilistic frameworks such as maximum likelihood and Bayesian inference as well as generative graph neural networks. Each method carries its assumptions and computational complexities, but collectively they offer a powerful toolkit to elucidate evolutionary scenarios. The inferred phylogenetic trees not only unravel the relatedness among species but also enable scientists to decipher patterns of speciation, understand the tempo and mode of evolutionary change, particularly in the realm of medicine. Phylogenetics aids in understanding the origins and evolution of diseases. It helps track the spread of pathogens, such as viruses and bacteria, enabling better disease management and the development of targeted treatments, vaccines, and antimicrobial strategies.

<a name="books_and_lectures" />

## Books and Lectures
- Harmon, L. 2019. [Phylogenetic comparative methods](https://lukejharmon.github.io/pcm/). (Version 1.4, released 15 March 2019). Published online by the author.
- Yang, Z. 2014. [Molecular evolution: a statistical approach.](http://abacus.gene.ucl.ac.uk/MESA/) Oxford University Press.
- Baum, D. A., and S. D. Smith. 2013. [Tree thinking: an introduction to phylogenetic biology.](https://www.amazon.com/Tree-Thinking-Introduction-Phylogenetic-Biology/dp/1936221160) Roberts and Company Publishers, Greenwood Village, Colorado.
- Garamszegi, L. Z. 2014. [Modern phylogenetic comparative methods and their application in evolutionary biology: concepts and practice.](https://link.springer.com/book/10.1007/978-3-662-43550-2) Springer-Verlag, Berlin.
- Hall, B. G. 2011. [Phylogenetic trees made easy: a how-to manual (4th edition).](https://www.amazon.com/Phylogenetic-Trees-Made-Easy-How/dp/0878936068) Sinauer Associates, Sunderland. (A guide to running some of the most important phylogenetic software packages.)
- Felsenstein, J. 2004. [Inferring phylogenies.](https://global.oup.com/ushe/product/inferring-phylogenies-9780878931774?cc=us&lang=en&) Sinauer Associates, Sunderland. (Comprehensive overview of both history and methods of phylogenetics.)

<a name="papers" />

## Papers

<a name="databases" />

## Real benchmark datasets
- Hedges SB, Moberg KD, Maxson LR. Tetrapod phylogeny inferred from 18S and 28S ribosomal RNA sequences and a review of the evidence for amniote relationships. Mol Biol Evol. 1990 Nov;7(6):607-33. doi: 10.1093/oxfordjournals.molbev.a040628. Erratum in: Mol Biol Evol 1991 May;8(3):398. PMID: 2283953.
- Garey JR, Near TJ, Nonnemacher MR, Nadler SA. Molecular evidence for Acanthocephala as a subtaxon of Rotifera. J Mol Evol. 1996 Sep;43(3):287-92. doi: 10.1007/BF02338837. PMID: 8703095.
- Ziheng Yang, Anne D. Yoder, Comparison of Likelihood and Bayesian Methods for Estimating Divergence Times Using Multiple Gene Loci and Calibration Points, with Application to a Radiation of Cute-Looking Mouse Lemur Species, Systematic Biology, Volume 52, Issue 5, 1 October 2003, Pages 705–716
- Daniel A Henk, Alex Weir, and Meredith Blackwell. Laboulbeniopsis termitarius, an ectoparasite of termites newly recognized as a member of the laboulbeniomycetes. Mycologia, 95(4):561–564, July 2003.
- Clemens Lakner, Paul Van Der Mark, John P Huelsenbeck, Bret Larget, and Fredrik Ronquist. Efficiency of Markov chain Monte Carlo tree proposals in Bayesian phylogenetics. Systematic biology, 57(1):86–103, 2008
- Ning Zhang and Meredith Blackwell. Molecular phylogeny of dogwood anthracnose fungus (discula destructiva) and the diaporthales. Mycologia, 93(2):355–365, 2001.
- Anne D. Yoder and Ziheng Yang. Divergence dates for malagasy lemurs estimated from multiple gene loci: geological and evolutionary context. Molecular Ecology, 13(4):757–773, 2004.
- Amy Y. Rossman, John M. McKemy, Rebecca A. Pardo-Schultheiss, and Hans-Josef Schroers. Molecular studies of the bionectriaceae using large subunit rdna sequences. Mycologia, 93(1): 100–110, 2001. 




<a name="methods" />

## Methods

<a name="computational_tools" />

## Computational Tools
- [Ultrafast Sample Placement on Existing Trees (UShER)](https://github.com/yatisht/usher): UShER is a package of a family of programs for rapid phylogenetic analyses, particularly suitable for the SARS-CoV-2 genomes. It can rapidly places new samples onto an existing phylogeny using maximum parsimony.
- [RAxML Next Generation](https://github.com/amkozlov/raxml-ng): RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML) optimality criterion.
 


